Modules/ChequeModule/Models/Cheque.cs

```diff
@@ -16,12 +12,12 @@ public class ChequeModel
 
         public DateTime DateCreated { get; set; }
 
-        public string AmountInWord { get; set; }
+        public string AmountInWord { get; set; } = string.Empty;
```

Comment:
My initial intention besides resolving the nullable exception, is that ```... = string.Empty;``` make sure it will not be a null value, to use ```string?``` makes it ok to be null. But I'm not sure will there be a difference for our use case la. So anyhow I already modified it to ```string?```.